examples/seismic/self_adjoint/time_blocking.ipynb

```diff
@@ -0,0 +1,1365 @@
+{
```

Comment:
<blockquote>Define symbols</blockquote><p> </p><p>Definition of symbols or Table of symbols</p><p>(?)</p><p> </p><blockquote>theJacobian</blockquote><p>the Jacobian</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
done